test/run-integration-tests

```diff
@@ -130,7 +124,7 @@ run-test-for-cluster-size() {
 }
 
 #
-# Run the integration tests against the testpop binary.
+# Run the integration tests against the testpop binary, in parallel.
```

Comment:
Add a "TODO: failing as of `date +%F`." Also, do we have an issue in the issue tracker?